packages/analytics-js-common/src/utilities/eventMethodOverloads.ts

```diff
@@ -23,7 +24,7 @@ export type TrackCallOptions = {
 
 export type IdentifyCallOptions = {
   userId?: string | null;
-  traits?: Nullable<ApiObject>;
+  traits?: Nullable<IdentifyTraits>;
```

Comment:
do group traits forllow the same object type?

Reply:
Not necessarily. It is about the group information where, firstName, dob doesn't make sense.